README.md

```diff
@@ -102,11 +118,14 @@ annotation to your .proto file
 
 3. Generate gRPC stub
 
+   Suppose `ROOT_TO_PROTO` is where your proto files store and `DEST` is where you want put generated files in, do:  
+
    ```sh
-   protoc -I/usr/local/include -I. \
+   protoc -I/usr/local/include \
+     -I${ROOT_TO_PROTO} \
```

Comment:
I don't know if this is helping tbh. `-I.` means current directory, not `path/to/your_service.proto`. I'd say remove the `ROOT_TO_PROTO` but keep the `DEST`.